EXTEND.rst

```diff
@@ -0,0 +1,176 @@
+Extending openwisp-ipam
```

Comment:
Ajay let's not divide the README.
In the future, when we'll improve the docs we may put effort in dividing the content in different page, but now let's just offer everything in one document for consistency with all the other modules (it's also better from an SEO standpoint in this case, as the first page of each repo has better ranking).

Reply:
Okay (I have to make that change it all the PRs)! :smile: